docs/tutorials/computers101.md

```diff
@@ -0,0 +1,248 @@
+# How to Computers
+!!! Overview 
+    This page will provide guidance and key points for navigating and operating with the Unix Terminal, Git, Python, and SSH. 
+    Concepts, useful command charts, and demonstration of function are below. See <a href="Software-Carpentry.org">Software Carpentry</a>
+    for in depth instruction. 
+
+<a name="toc"></a> **Jump to:**
+- [How to Terminal](#terminal)
+- [How to `git` and GitHub](#git)
+- [How to `python`](#python)
+- [How to SSH](#ssh)
+
+---
+
+## <a name="terminal"></a> How to Terminal
+!!! note "Summary" 
+    See <a href="Software-Carpentry.org">Software Carpentry</a> for instruction. Below are key take aways and what to look for in the link. 
+    We will be using the Unix Shell to:
+    <blockquote> 
+    -navigate files and directories
+    <br>-work with files, directories, loops, and scripts
+    <br>-utilize pipes and filters 
+    <br>-find strings, files, and directories that match a pattern
+    <br>-learn how to combine commands to carry out and achieve tasks 
+    <br>-learn shortcuts for ease of working with a programming language
+    </blockquote>
+ 
+<br>**The Essentials**
+<br><br>**The Shell** 

```

Comment:
Similar to the previous comment, use the hierarchical heading levels instead. See https://www.markdownguide.org/basic-syntax/#headings